samples/azure-quantum/entanglement-swap/ES-quantinuum-qiskit.ipynb

```diff
@@ -0,0 +1,930 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "all_results = dict()\n",
+    "\n",
+    "def get_fidelity(counts):\n",
```

Comment:
In line with the comment above, measuring in a single basis is not sufficient to conclude a fidelity such that I would strongly suggest renaming here.